tests/jenkins/Jenkinsfile

```diff
@@ -132,6 +132,13 @@ pipeline {
                         }
                     }
                 }
+                stage('nrntraub'){
+                    steps{
+                        dir('nrntraub'){
+                            git branch: 'icei', url: 'https://github.com/iomaganaris/nrntraub.git'
```

Comment:
I am trying to minimise forks. Could you push your changes to my repo as well? (Ideally we want to have everything under neuronsimulator soon).